### DISCUSSION TITLE
Anti Harassment Policy - Creating a Plan of Action for Event Organisers

### DISCUSSION THREAD

**Discussion by SQLDBAWithABeard** [dataplat/DataSaturdays#288] [General]
Following a number of twitter DMs and other conversations with event organisers, I have created a sample Meeting Agenda to help organisers to create a plan of action and include all relevant points.

You can find it here

https://github.com/dataplat/DataSaturdays/blob/AHP-plan/docs/AHP-Plan-Sample-Meeting-Agenda.md 

Feel free to update this, you will need to fork and create a PR - Its not scary, GitHub will do it all for you - https://docs.github.com/en/repositories/working-with-files/managing-files/editing-files#editing-files-in-another-users-repository

This discussion and the meeting agenda and the plan are about your Anti Harassment Policy - not about *a ban list of offenders* That is a separate discussion with its own unique challenges

## Older Description

It is really important that there is an opportunity to hear viewpoints from all of the people.

For those who wish to do a thing right now because their event is coming up. I advise this (correct me if I am wrong) - 

- Believe what trusted people tell you, there is no need for you to have details to decide "how bad it is"
- Have a Code of Conduct in public and require ticket holders to agree to it
- Have an Anti Harassment Policy and require ticket holders to have read it
- Discuss and write down in private a plan of action for you the organisers to take should you be required to respond to an incident.  This will ensure that all organisers are in agreement. That things have been decided whilst emotions are not high. That there is a list of things to follow like a DR checklist to bring it back to work parlance. 
Things that have been mentioned in public that should be considered
       - Who will support the person being harassed, where will this happen?
![image](https://user-images.githubusercontent.com/6729780/159688305-d2b11c09-ad5a-448f-9069-d78f8262248f.png)
       - Who will be the person who is the voice of the event to the harasser
               - ![image](https://user-images.githubusercontent.com/6729780/159687398-88f5b9b0-005b-415e-940a-261135028b32.png)
![image](https://user-images.githubusercontent.com/6729780/159687454-3fb013d1-6330-4e97-9f62-963aa3393dda.png)
                 https://twitter.com/TheStephLocke/status/1506365604214759430?s=20&t=8NEsEFgyc555ziDygvfuaA 
       - How will re-entry be denied
       -  venue security?  Police? Big beardy person? 
       - How will you support the harassed person post event, at adjoining social events in public places
       -

**Comment by way0utwest** [comment]
For my, or other, reference https://www.cathrinewilhelmsen.net/why-your-event-needs-code-of-conduct-and-plan-process-for-enforcing-it/

My view, as the person trying to run SQL Saturday, is that we ought to provide some specific guidance on how to handle a report, take statements, and what actions an organizer should take. Certainly local laws are important here as well, and I don't know how to deal with those, other than ask the community to provide their own local updates to any advice or policy.

I upset, so this might not be how I feel later, but I am thinking that my guidance would be to:
- act earlier rather than later. 
- I'd prefer that an organizer bias themselves towards a victim over a harasser
- make a bad decision and apologize over inaction.

I am trying to think of this is a man accusing a woman as well as a woman accusing a man.

**Comment by tameraclark** [comment]
I worked with a few organizers to face issues as a team. Of course onsite decisions were going to be needed but then the "now what do we do" could be a committee discussion.  

We have had many discussions on how do we report and track activity. Someone has to be the gate keeper.

**Reply by way0utwest** [reply]
Would love to see you contribute a doc with specifics on how you have, or how you recommend, someone handle this.

**Comment by SQLDBAWithABeard** [comment]
Let's not be contributing on docs yet please @way0utwest 

Is more important that we listen to everyone's views and they can give them in public and everyone can listen. 

I have DMs full of opinions and the difficulties. 

For now I would encourage that there is just a discussion please.

**Comment by SQLDBAWithABeard** [comment]
I think it's important there is a plan written down by the organisers of how they will take action.  So that it is discussed when all is calm and each member of the organisation can agree and then there is something to follow when emotions run high

**Reply by tameraclark** [reply]
We have our event at a university. The plan includes our sponsor (the department head), the campus police, a male and female representative that can speak to someone if they have a problem, an empty room for privacy, a form just to pencil down needed information and then a post event discussion as to how big this is/where to go from there.  We've not needed it thus far.  

I started a new session just before the pandemic hit. "The community speaks" This was a place to gather feedback to see what attendees wanted event organizers to do, how to justify a punishment, how to protect everyone, methods for deflecting bad situations and the list went on. I got so much out of these sessions. So many great discussions. Valuable input.

**Comment by SQLDBAWithABeard** [comment]
I also have strong views on any sort of requirement for the person who has faced the harassment to be the one who should inform anybody else for later events.  The word of a trusted person should be enough

**Reply by potatoqualitee** [reply]
Strongly agreed. If it came down to it and I had to talk about my trauma over and over with untrusted parties, I'd just stop going to conferences or limit my attendance to conferences with organizers that are personal friends.

> I told the organizers all of this, but I shouldn’t have had to. I told the organizers all of this, and some of them _still_ weren’t sure what to do because nothing had actually happened _at the event_.

This response, along with a number of other common refrains, would greatly pile on to my distress. The option for someone else with more emotional bandwidth to deal with this situation on my behalf is necessary.

**Reply by Nullh** [reply]
Absolutely! We shouldn't require the receipts to believe someone, and we shouldn't have to put the weight of notifying solely on the person affected.

**Comment by SQLDBAWithABeard** [comment]
There are difficulties with gatekeeping that need to be discussed

**Reply by Nullh** [reply]
Sorry Rob, could you explain this a bit?

**Reply by SQLDBAWithABeard** [reply]
This is from the earlier discussion. Where event organisers wanted lists of known abusers. 
Who polices that?
Who decides who is on the list?

This was the gatekeeping I meant here. However the perceived legal difficulties mean that I think the whisper campaign will stay for a while longer.

**Comment by kevchant** [comment]
I have been reading through all this and it reminded me of the popular 'Ask for Angela' scheme and it has made me wonder if something like that would be useful?

https://en.wikipedia.org/wiki/Ask_for_Angela

**Reply by SQLDBAWithABeard** [reply]
This is super good, thanks for adding it to the agenda

**Comment by dmaloo** [comment]
Other than what you're already suggested - taking a trusted person's word and what Steve says - a victim bias is preferable, am trying to think what further we can do and don't really know. That said, I believe there are issues with just those norms too. I have personally seen women misbehave at the summit too, for example, and was strongly recommended against not reporting them. I did not want too myself either since they were incapable of physical harm, just behaving poorly under influence of alcohol..but said victims did not think that was fair. There are many gray areas like this that need more nuance. Thank you for initiating this discussion. Mala

**Comment by datarelish** [comment]
I already created a lot of material but it wasn't adopted. Despite having had such bad experiences as a Microsoft Most Valuable Professional (MVP) and Microsoft Regional Director myself, I'm happy for the material to be reused to support other people so others don't have to go through this trauma and honestly I'm lucky I survived all the trauma that I was put through. 
I have moved on to kinder communities and I'd suggest looking at what Tableau does, for example. 
It is also worth noting that people can have a plan in place, but decide not to follow it because of 'stack ranking' i.e. one person is perceived as more influential than another person. 
Here are the links:

[Diversity and Inclusion Toolkit - includes other protected characteristics](https://github.com/datarelish/DiversityAndInclusionToolkit )
[Decency Charter](https://jenstirrup.com/2017/11/26/open-source-decency-charter-proposal-for-dealing-with-harassment-at-technical-events/)
[Decency Charter for Technical Communities](https://jenstirrup.com/2018/08/21/decency-charter-for-technical-communities/)
[Isn’t it time community speakers were all treated reasonably and equally by organizers?](https://jenstirrup.com/2021/12/14/isnt-it-time-community-speakers-were-all-treated-reasonably-and-equally-by-organizers/)
[Intersectionality, Diversity, Inclusion and Equality – how they can work together for workplace and tech community](https://jenstirrup.com/2021/09/04/intersectionality-diversity-inclusion-and-equality-how-they-can-work-together-for-workplace-and-tech-community/) 
[Increasing Intersectionality at community events](https://jenstirrup.com/2021/07/26/increasing-intersectionality-at-tech-community-events/)

**Reply by SQLDBAWithABeard** [reply]
Thanks for those comments and links Jen. I am sure that event organisers can use them to improve their events.

Having a plan written down makes it much easier for event organisers to know what to do "if" rather than being blindsided or caught unawares.

All event organisers should be aware of the possibility of the plan not being followed and I would encourage them to act accordingly

**Reply by SQLDBAWithABeard** [reply]
Your experiences are horrific, and I am sorry that you had to live them and continue to live them. They include many things that event organisers need to understand and consider when writing their plan.

This GitHub discussion has nothing to do with the MVP program or getting renewals. I think my contributions as a whole will be enough for Microsoft to decide if I deserve that recognition. I will not even put this in my activities. 

This discussion is to help event organisers to create a plan, hopefully by providing some framework for them. It includes thoughts, ideas and information from many people, some of them are MVPs of course, some are not, some work for large corporations, some do not wish to be public. I am proud and glad that this bearded face is able to enable people to collaborate on something that will make improvements to future events and I shall continue to be an ally and enabler.

**Reply by eugman** [reply]
Agreed with everything Rob just said. Also, thank you for keeping the conversation on target, Rob. These conversations are difficult at a base level and inevitably become more difficult because they involve personal harms and traumas.

**Reply by datarelish** [reply]
oh! I apologise you thought that comment was directed at you, Rob; it absolutely was not. I am glad of your allyship. Your voice carries more as a male and we can see that here. I am sharing this with you in the hope that it will support you in some way since I was not listened to, and perhaps the body of work I created will help you to help others.

**Comment by tameraclark** [comment]
So something I started doing at my UG meetings was after the welcome/open remarks I decided to talk about items that could be discussed in a few minutes. Now mind you covid hit and I only got to discuss Micro Aggressions. My plan had been to try and take tiny bites of what is uncomfortable and make it easier to talk about. 

One thing I worry about is that in the US we don't have very good laws that protect humans from harassment. Being a creep is also not against the law.  Yesterday in this house we were discussing this and I asked "So you are asking Walmart to kick someone out because he makes me feel uncomfortable?"  I just want to be sure as Steph has pointed out that the organizer of an event is "protected". Is there language that needs to be included in our CoC, our attendance policy and etc? 

I am also very interested in how we allow someone to redeem themselves.

**Reply by datarelish** [reply]
Very nice idea! I like the idea of achievable chunks where people have a safe space to have an introduction to these difficult questions. I also suspect that technical people want to 'do' something and sometimes they don't know what to 'do', so it is hard to get started. This feels like something that people could make a start to 'do' something positive.

**Reply by tameraclark** [reply]
Yes, they do want to 'do' something but don't know-how. In my session, great ideas were given as sort of canned responses to activities and it was very helpful. My favorite was to ask questions like "I am sorry, I don't understand what you mean".

**Reply by datarelish** [reply]
Love this, Tamara! Some of the issues involve people being caught off-guard momentarily and it helps people to put some thought into what they will do or say because they've had the time to reflect on it in advance. So, say, for example, you hear a man say something untoward to a female; what do you say? I have an answer prepped but not everyone does, and usually that means people go silent and that makes the victim feel ignored and unheard. I believe this is a really worthwhile venture, Tamara - great idea!
